zenodo/modules/search_ui/static/templates/zenodo_search_ui/facets.html

```diff
@@ -6,15 +6,15 @@
   </span>
 </a>
 <div ng-init="tree=[];tree_more=[]" ng-repeat="(key, value) in vm.invenioSearchResults.aggregations track by $index">
-  <div ng-if="key!=='communities'">
+  <div ng-if="key!=='publication_date'">
     <div ng-if="value.buckets.length > 0 || getValues(key)" class="panel panel-default">
       <div class="panel-heading">
         <h3 class="panel-title titlecase">{{ key|titlecase}}</h3>
       </div>
       <div class="panel-body">
         <ul class="list-unstyled" ng-repeat="item in value.buckets">
           <li class="titlecase">
-            <input type="checkbox" ng-checked="handler[key].indexOf(item.key) > -1" ng-click="handleClick(key, item.key)" /> {{ item.key|titlecase }} ({{ item.doc_count }})
+            <input type="checkbox" ng-checked="handler[key].indexOf(item.key) > -1" ng-click="handleClick(key, item.key)" /> {{ item.title || item.key|titlecase }} ({{ item.doc_count }})
```

Comment:
```suggestion
            <input type="checkbox" ng-checked="handler[key].indexOf(item.key) > -1" ng-click="handleClick(key, item.key)" /> {{ (item.title || item.key)|titlecase }} ({{ item.doc_count }})
```

I can't even with how Angular filters work, so better play it safe I guess ¯\_(ツ)_/¯